src/runtime/manifest-parser.pegjs

```diff
@@ -1497,6 +1497,7 @@ SchemaInlineField
       type
     });
   }
+  / '*'
```

Comment:
This seems to me will break the type declaration of the AST nodes. Concretely we today declare that:

```
export interface SchemaInline extends BaseNodeWithRefinement {
  kind: 'schema-inline';
  names: string[];
  fields: SchemaInlineField[];
}
```

But with this change the `SchemaInline.fields` could have an element that is a string `'*'`.

Reply:
Is it alright to make `type` nullable -- or to simply pass a null value here? I went down this road, but I created a new type for `SchemaType` and then opted against it.

Reply:
Could `TypeName` work?

Reply:
Reading more pegjs code, it seems like a common pattern to null out certain fields, including type.

Reply:
Yup, null types get filled in when the ast is augmented with type information in `manifest.ts`